### PR TITLE
Fix some flaky CI tests

### PR DESCRIPTION
**Pull request by davidvarga93**
refs: MBL-15621
affects: Teacher, Student
release note: none

test plan:

### CODE REVIEW

**Comment by davidvarga93** [comment]
Still WIP, checking nightly: https://app.bitrise.io/build/c3db5652-62b3-4fd5-a42e-9fef86322495#?tab=log

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 91.81% | 91.81% | 0%

[MBL-15621](https://instructure.atlassian.net/browse/MBL-15621)

#### Affected Apps: Teacher, Student
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/91036940/p/454b34325bf655ddd44ce1d12a0c986e">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/91036943/p/7e76f7675d444357f1183e4da51e5319">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/91036939/p/84204166d144d3413401edb3f0963d76">
    
  </a>
</details>

File: TestsFoundation/TestsFoundation/CoreUIElements/K5/K5Schedule.swift [hunk 1]
```diff
@@ -20,4 +20,20 @@ import XCTest
 
 public enum K5Schedule: ElementWrapper {
     case today
+
+    public static var previousWeekButton: Element {
+        app.find(label: "Previous Week", type: XCUIElement.ElementType.button)
```
Comment (vargaat) [inline TestsFoundation/TestsFoundation/CoreUIElements/K5/K5Schedule.swift:25]: In such cases the type of the `type` parameter is defined in the `find()` method signature so we can omit `XCUIElement.ElementType` and only write `.button`.

**Comment by davidvarga93** [comment]
Still WIP, another iteration of nightly:
https://app.bitrise.io/build/8d5bb637-8efb-48c9-961e-83af2d3b6b67#?tab=log

**Review by vargaat — APPROVED** [review]

**Review by balintbartok — APPROVED** [review]

**Pull request merged**